### PR TITLE
Test to Reproduce JIRA_10041 Exception

### PR DESCRIPTION
**Pull request by sylvestor88**
Kill C\* on any one of the node while the application is running. This will reproduce the error and the test will finish as a pass.

### CODE REVIEW

**Comment by ptnapoleon** [comment]
How is InsertData any different than SimpleWrite? Why do we need a whole new module

File: test/validation/org/apache/cassandra/modules/UpdateCounterModule.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            Cluster cluster = Cluster.builder().addContactPoints(bridge.clusterEndpoints()[0]).build();
+            Session session = cluster.connect();
+            session.execute("USE k");
+            session.execute("CREATE TABLE c (id int PRIMARY KEY, mycounter counter)");
+            PreparedStatement update = session.prepare("UPDATE c SET mycounter=mycounter+1 WHERE id=1");
+            Statement select = QueryBuilder.select().all().from("t").limit(100);
+            update.setConsistencyLevel(ConsistencyLevel.QUORUM);
+            boolean exception = false;
+
+            while(exception == false)
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/modules/UpdateCounterModule.java]: Better to make this `while(!exception)`

File: test/validation/org/apache/cassandra/modules/UpdateCounterModule.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                            exception = true;
+                            logger.error(e.getMessage());
+                        }
+                    }
+                }
+            }
+
+            try
+            {
+                Assert.assertEquals(true, exception);
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/modules/UpdateCounterModule.java:90]: Am I correct in reading that this module is supposed to run into errors?

**Comment by ptnapoleon** [comment]
You mention that C\* needs killed to reproduce the error, but where is that happening?

File: test/validation/org/apache/cassandra/modules/UpdateCounterModule.java (hunk 1 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/modules/UpdateCounterModule.java]: Done

File: test/validation/org/apache/cassandra/modules/UpdateCounterModule.java (hunk 2 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/modules/UpdateCounterModule.java:90]: Yes. As per the ticket I wanted to check if the error occurs in the given scenario. Therefore, I made the test pass if it does.

**Comment by sylvestor88** [comment]
I am doing that manually, by kill -9  which produces the error and the test is passed.

**Comment by sylvestor88** [comment]
Well InsertDataModule is little different from SimpleWriteModule.
i) replication_factor
ii) No. of Inserts
iii) No Assertion
I could have made changes to SimpleWriteModule but then thought of rather making a different module as CVH allows to create any number of modules. I can edit the SimpleWriteModule if that's a better way to go about it.

**Pull request closed without merging**